.hemtt/base.toml

```diff
@@ -0,0 +1,12 @@
+name = "Dynamic-Camo-System"
```

Comment:
Add ``modname = "Dynamic Camo System"`` so the name of the addon stays consistent with what it was before. If the name of an addon changes and it's published to the workshop, Steam will download the addon again under the new name, essentially creating a duplicate.